src/NNlibCUDA.jl

```diff
@@ -4,6 +4,9 @@ using NNlib
 using CUDA
 using Random, Statistics
 
+const IntOrIntTuple = Union{Integer, NTuple{N,<:Integer} where N}
+const MAX_THREADS = 1024
```

Comment:
this isn't safe, querying the occupancy API would be better here

Reply:
I modified it. Is it good now?